cli/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
-name = "apollo"
-version = "0.0.1"
-authors = ["jbaxleyiii <james@apollographql.com>", "ashi <ashi@apollographql.com>"]
+name = "ap"
```

Comment:
thinking about this, there's no reason we have to rename it here, right? We could name this `cli` or `apollo` or `apollo-cli`?

Reply:
so, here's the thing… it seems like for the installed binary, we maybe want to go with `ap` but also sortof want to keep `apollo` if we can, and i'm still holding out a little bit of hope for `a`/`an`. in any case, i think it's safe to say that the final installed command name isn't set in stone.

So maybe the crate shouldn't be named the final installed binary, but instead `apollo-cli` or something that will definitely not be the final name of the installed command, but is the name of the crate.

This way, we don't have to change the tests and scripts as we iterate on command names. You'll always do `cargo run --bin=apollo-cli`, the tests can always call `Command::cargo_bin("apollo-cli")` and so on

Reply:
@queerviolet ah I see what you are getting at now. Sure thing! I'll make the adjustments. Do you prefer `apollo-cli` or just `cli`?